app/views/partials/_header.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-expand-lg bg-body-tertiary  bg-dark border-bottom border-body" data-bs-theme="dark">
+  <div class="container-fluid">
+    <%= link_to "BlogApp", root_path, class:"navbar-brand"%>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <ul class="navbar-nav me-auto mb-2 mb-lg-0  nav justify-content-end" >
+      <li class="nav-item">
+        <%= link_to "Users", users_path, class:"nav-link"%>
+      </li>
+      
+      <li class="nav-item">
+        <%= link_to "Login", new_user_session_path, class:"nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Sign Up", new_user_registration_path, class:"nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Logout", users_sign_out_path, class: "nav-link" %>
+      </li>
+      
```

Comment:
- [OPTIONAL ] Great job adding the logout link as well but I could see it on the same page for logging in or signing up I think here is useless since if the user didn't log into the app, So how he can log out? so I recommend adding it after the user logs into the app 💯🚀🚀


![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/d92a4021-b755-46bb-9f42-3f1c253589c8)